pkg/controller/statefulset/stateful_set_test.go

```diff
@@ -50,8 +52,9 @@ func alwaysReady() bool { return true }
 
 func TestStatefulSetControllerCreates(t *testing.T) {
 	set := newStatefulSet(3)
-	ssc, spc, om, _ := newFakeStatefulSetController(set)
-	if err := scaleUpStatefulSetController(set, ssc, spc, om); err != nil {
+	_, ctx := ktesting.NewTestContext(t)
```

Comment:
```suggestion
	logger, ctx := ktesting.NewTestContext(t)
```


Then below you don't need klog.FromContext.

Same in other tests.